tasks/style.py

```diff
@@ -24,7 +24,7 @@ def black_check(ctx):
 @task
 def isort_check(ctx):
     """Check style through isort"""
-    ctx.run(f"{VENV_PREFIX} isort --atomic --apply --check-only")
+    ctx.run(f"{VENV_PREFIX} isort --atomic --check-only .")
```

Comment:
This syntax seems to work only on isort >= 5.0.0. I'd suggest pining an isort version higher than that.

Reply:
Indeed. I updated isort to follow more recent isort tool. Pinning will be a good idea.